package.json

```diff
@@ -44,33 +55,30 @@
   },
   "scripts": {
     "start": "node ./scripts/start-dev-server.js",
-    "setup": "bash ./scripts/setup.sh",
+    "setup": "npm ci && npx husky install",
     "clean": "find . -name \"node_modules\" -type d -prune | xargs rm -rf",
     "clean:dist": "find ./packages -name \"dist\" -type d -prune -maxdepth 2 | xargs rm -rf",
     "clean:tests": "node ./scripts/cleanup-test-results.js",
-    "bootstrap": "lerna clean && lerna bootstrap",
     "dist:flat": "gulp dist:flat",
     "dist:swatches": "gulp dist:swatches",
-    "sass": "sass-build",
-    "sass:flat": "npm run dist:flat && sass-build --config sass-flat.config.js",
-    "sass:swatches": "npm run dist:swatches && sass-build --config sass-swatches.config.js",
-    "sass:standalone": "lerna run sass:standalone",
+    "sass": "npm run sass -ws --if-present",
+    "sass:flat": "npm run sass:flat -ws --if-present",
+    "sass:swatches": "npm run sass:swatches -ws --if-present",
+    "sass:standalone": "npm run sass:standalone -ws --if-present",
     "build": "npm run sass && npm run build:scripts",
     "build:scripts": "npm run build --prefix packages/html",
     "build:tests": "npm run build:tests --prefix packages/html",
     "eslint": "eslint \"**/*.{js,jsx,ts,tsx,mjs}\"",
     "check-types": "npm run check-types --prefix packages/html",
     "stylelint": "stylelint \"**/*.scss\"",
-    "html-validate": "html-validate tests",
     "lint": "npm run eslint && npm run stylelint",
     "lint-staged": "lint-staged --verbose",
-    "docs": "lerna run docs",
-    "docs:check": "npm run docs && lerna exec --ignore @progress/kendo-themes-html -- git diff --exit-code --quiet -- docs/",
+    "docs": "npm run docs -ws --if-present",
+    "docs:check": "npm run docs -ws --if-present -- git diff --exit-code --quiet -- docs/",
     "render-test-pages": "node ./scripts/render-test-pages.mjs",
     "create-screenshots": "./build/create-screenshots.sh",
     "embed-assets": "gulp assets",
     "create-component": "gulp create-component",
-    "update": "npx npm-check-updates --upgrade",
```

Comment:
You will be surprised, but this bit is useful.

Reply:
Removing in favor of dependabot ( grouped updates and versioning strategy ).